modules/custom/az_core/az_enterprise_attributes_import/tests/src/Functional/EnterpriseAttributesImportTest.php

```diff
@@ -0,0 +1,105 @@
+<?php
+
+namespace Drupal\Tests\az_core\Functional;
+
+use Drupal\Tests\BrowserTestBase;
+use Drupal\migrate\MigrateMessage;
+use Drupal\migrate_tools\MigrateExecutable;
+
+/**
+ * Test of attribute import functionality.
+ *
+ * @ingroup az_enterprise_attributes_import
+ *
+ * @group az_enterprise_attributes_import
```

Comment:
@tadean Just curious: are both of these annotations needed?

Also, I see that Drupal 11 supports tests that use attributes in place of annotations: https://www.drupal.org/node/3447698

Reply:
I've removed the superfluous group declaration, it doesn't seem to be needed. Thanks @bberndt-uaz !

I also tried out the attribute version, but it does look like it's restricted to PHPunit 10 in D11.